modules/optimization/src/executioners/AdjointSolve.C

```diff
@@ -52,6 +52,10 @@ AdjointSolve::solve()
   if (_inner_solve && !_inner_solve->solve())
     return false;
 
```

Comment:
add a comment please

this shouldnt be necessary, but it is so let s remember why

Reply:
I don't understand; this isn't part of this PR.

Reply:
the three lines below are?

Reply:
This is what fixedPointSolve.C does when the inner_solve fails and then the return false is to catch when this solverObject fails in SteadyAndAdjoint.C

But this is not part of Oana PR.

Reply:
I ll need to be more accurate with my clicks. One line too high and 3 people had to get involved